kitsune/sumo/static/sumo/scss/base/forms/_fields.scss

```diff
@@ -411,5 +411,10 @@ textarea {
 label.required::after {
   content: "\A Required";
```

Comment:
The `\A` line break really adds a lot of space, just before the `Required` text, which to me looks off:
<img width="626" alt="image" src="https://github.com/mozilla/kitsune/assets/3743693/04b6720c-271c-495b-ad54-3734632d24c3">

compared to the other fields:
<img width="367" alt="image" src="https://github.com/mozilla/kitsune/assets/3743693/c6d2e9db-d6bd-4f77-8e29-f5b4969a065e">

I think it looks better with the `\A` removed, but because the `Required` text is added by this CSS, it's really difficult to position the same as the label.
<img width="350" alt="image" src="https://github.com/mozilla/kitsune/assets/3743693/33788c78-e3b4-4a75-a59c-eb637b423c32">

Reply:
Thanks for pointing out - I've tightened it up a bit.